src/Pages/ContactPage.js

```diff
@@ -21,11 +25,55 @@ class ContactPage extends React.Component {
     return (
       <div>
         <NavBar />
-        <Container className="contactUs">
-          <h1>Have any questions?</h1>
-          <h2>Send us an email</h2>
-          <ContactForm />
-        </Container>
+        <div class="bg-dark"></div>
+        {/******WORD******/}
+        <div className="contactWordLocation">
+          <container className="contactWord">
+            <h1 class="display-3">Have any questions ?</h1>
+            <br></br>
+            <h2 class="display-5">we are happy to help <span class="display-1">:)</span></h2>
+
+
+            <br></br>
+            <br></br>
+            <br></br>
+
+            <img src="C001Email.svg" alt="Email-Logo" width="45"></img>
+            <span className="contactInfo">Email: ############@######</span>
+
+
+            <br></br>
+            <br></br>
+            <br></br>
+
+            <img src="C002WeChat.svg" alt="Wechat-Logo" width="45"></img>
+            <span className="contactInfo">Wechat ID: ############</span>
+
+
+            <br></br>
+            <br></br>
+            <br></br>
+            <br></br>
+            <br></br>
+            <img className="imgmargin" src="C003Facebook.svg" alt="Facebook-Logo" width="50"></img>
```

Comment:
For components without anything inside, you can do  `<img className="imgmargin" src="C003Facebook.svg" alt="Facebook-Logo" width="50" />` instead for cleaner coding style.